13-RemoteRepositorySetup/README.md

```diff
@@ -0,0 +1,75 @@
+# Working with remote repository
+
+## Introduction
+
+* So far we worked on local repository.
+* In real world you would have a remote copy for several reasons:
```

Comment:
I kept first would because it is meant as willingness, re-stated second sentence on line 9